app/data/models/generation.js

```diff
@@ -3,7 +3,8 @@ module.exports = (sequelize, DataTypes) => {
     generationId: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
     statementData: DataTypes.JSON,
     dateGenerated: DataTypes.DATE,
-    filename: DataTypes.STRING
+    filename: DataTypes.STRING,
+    documentReference: DataTypes.STRING
```

Comment:
We will be conducting search on `documentReference`, hence, there must be a !nullable constrain on it. We need to update the new field for all existing records before applying the constrain.